src/index.ts

```diff
@@ -84,6 +85,35 @@ async function main() {
                 const data = await getMarkerprice('ram')
                 res = Response.json(data)
             }
+            if (path === '/api/rex/total_lent') {
+                const data = await getREX('total_lent')
+                res = Response.json(data)
+            }
+            if (path === '/api/rex/total_unlent') {
+                const data = await getREX('total_unlent')
```

Comment:
What is `total_unlent`? 😄

Reply:
My first thought was that it was the total lendable, but we already have that value below.

Reply:
I don't know either, lol